### DISCUSSION TITLE
Turn off  reader mode for all

### DISCUSSION THREAD

**Discussion by plundy322** [lwindolf/liferea#1199] [Q&A]
How to turn off reader mode for all listings by default?
Thanks, P.L.

**Comment by mhoran** [comment]
Tools -> Preferences, uncheck "Use Reader mode".

![Screenshot_20230120_165926](https://user-images.githubusercontent.com/5330/213812836-51e4c9cf-3fde-4716-b705-b858e69d50bd.png)

**Reply by plundy322** [reply]
Thank you!
